components/ldap.rst

```diff
@@ -0,0 +1,66 @@
+.. index::
+   single: Ldap
+   single: Components; Ldap
+
+The Ldap Component
+==================
+
+    The Ldap component provides a means to connect to an LDAP server (OpenLDAP or Active Directory).
```

Comment:
It sounds weird to use an "a" with a plural, shouldn't it be "provides the means"?